Apps/Playground/iOS/LibNativeBridge.mm

```diff
@@ -98,8 +97,8 @@ - (void)render
 {
     if (graphics)
     {
-        graphics->FinishRenderingCurrentFrame();
         graphics->StartRenderingCurrentFrame();
+        graphics->FinishRenderingCurrentFrame();
```

Comment:
Note that this will change the behavior of rendering a bit. The idea with calling in this order (Finish then Start) is that Finish will block and thus we want to give the app the most amount of time to do other things. @syntheticmagus Any thoughts?